homeassistant/helpers/intent.py

```diff
@@ -146,6 +152,7 @@ def __init__(
         self,
         name: str | None,
         area: str | None,
+        floor: str | None,
```

Comment:
Any reason we don't have default values for these params ?

Reply:
Nope, I'll add them.